blogposts/2020/3.19/lsif-go.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "lsif-go improvements"
+author: Eric Fritz
+publishDate: 2020-08-17T08:00-07:00
+tags: [blog]
+slug: todo
+heroImage: /blog/3.19-todo.png
+published: true
+---
+
+The release of Sourcegraph 3.19 happens to coincide with the [one-point-oh release](https://github.com/sourcegraph/lsif-go/releases/tag/v1.0.0) of [lsif-go](https://github.com/sourcegraph/lsif-go). This release boasts impressive speed improvements that brings intelligence to your Go code faster than ever before.
+
+The code intelligence is hard at work trying to bring the _time to intelligence_ as close to zero as possible. We want your code to always have up-to-date precise code intelligence on the newest commits to help aid users browsing unfamiliar code and code reviewers trying to understand new code in pull requests. Indexing speed is a _huge_ part of this, and we've made a great leap here.
+
+As an illustrative example, checkout the visual difference between indexing our [main repository](https://github.com/sourcegraph/sourcegraph) with lsif-go v0.9 and indexing it with lsif-go v1.0.
+
+#### Old and busted (v0.9.0)
```

Comment:
I suggest to use a different adjective here then "busted" - busted implies broken and I don't think we want to indicate that lsif-go was broken, just much slower. I suggest something like "sluggish" or even better to appear less "old" friendly - to match the title "the new hotness" with a similar paraphrase "the slow dinosaur" or something like that.

Reply:
for reference: https://www.youtube.com/watch?v=ha-uagjJQ9k

might still be worthwhile to change though :stuck_out_tongue:

Reply:
You are way cooler then me Garo, I'm just all textbook nerd 📖